torch/csrc/jit/ir.h

```diff
@@ -965,6 +965,10 @@ struct Graph {
 
   Graph() : Graph(c10::make_intrusive<Scope>()) {}
 
+  size_t size() const {
```

Comment:
I'd like to know if I can get around this with the current API

Reply:
Where is it being used? Lots of methods are called size so it hard to tell.

Reply:
used here https://github.com/pytorch/pytorch/pull/17100/files#diff-3da370a10cc2eae9761ef9968ee979ceR216

Reply:
For now, maybe just avoid the hack by putting the following in the matcher code and leaving ir.h unchanged:

```
size_t v = 0, n = 0;
for(Node* i : graph.nodes()) {
  v += i->outputs().size();
  n++;
}
```
The matcher is going to traverse the whole graph multiple times anyway so it won't add much overhead.